index.html

```diff
@@ -1 +1,15 @@
-hello, world
+<!DOCTYPE html>
```

Comment:
@AleksandarVucenic this looks great for now, maybe we will need to update it when we decide what technology stack we will use.

Reply:
Yes it will be updated leather this init html